<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>51</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nimi</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>71</y>
      <width>125</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Kirjoita nimesi kenttään</string>
    </property>
    <property name="toolTipDuration">
     <number>3</number>
    </property>
    <property name="placeholderText">
     <string>Kirjoita nimesi</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="heightDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <double>300.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="weightDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <double>300.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mittauspäivä</string>
    </property>
   </widget>
   <widget class="QLabel" name="heightLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>50</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Pituus</string>
    </property>
   </widget>
   <widget class="QLabel" name="weightLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>50</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Paino</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculatePushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Laske</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savePushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tallenna</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sexComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Valitse sukupuoli</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="neckSpinBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="waistSpinBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="hipsSpinBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QLabel" name="sexLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>70</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Sukupuoli</string>
    </property>
   </widget>
   <widget class="QLabel" name="neckLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>70</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Kaula</string>
    </property>
   </widget>
   <widget class="QLabel" name="waistLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>70</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Vyötärö</string>
    </property>
   </widget>
   <widget class="QLabel" name="hipsLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>70</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Lantio</string>
    </property>
   </widget>
   <widget class="QLabel" name="birthDateLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Syntymäpäivä</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="birthDateEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>80</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Painoindeksi</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>340</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="bmiLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Painoindeksi</string>
    </property>
   </widget>
   <widget class="QLabel" name="fatFiLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rasvaprosentti(FI)</string>
    </property>
   </widget>
   <widget class="QLabel" name="bmiLabel_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>100</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Rasvaprosentti(FI)</string>
    </property>
   </widget>
   <widget class="QLabel" name="fatUsLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rasvaprosentti(US)</string>
    </property>
   </widget>
   <widget class="QLabel" name="bmiLabel_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>110</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Rasvaprosentti(US)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>383</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>birthDateEdit</tabstop>
  <tabstop>heightDoubleSpinBox</tabstop>
  <tabstop>weightDoubleSpinBox</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>sexComboBox</tabstop>
  <tabstop>neckSpinBox</tabstop>
  <tabstop>waistSpinBox</tabstop>
  <tabstop>hipsSpinBox</tabstop>
  <tabstop>calculatePushButton</tabstop>
  <tabstop>savePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
